data/mods/Magiclysm/items/comestibles.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "owlbear_egg_stone",
+    "name": "owlbear egg",
+    "weight": 2395,
+    "color": "gray",
+    "spoils_in": "14 days",
+    "comestible_type": "FOOD",
+    "symbol": "0",
+    "quench": 128,
+    "healthy": 1,
+    "calories": 2560,
+    "description": "This egg laid by an owlbear seems almost indistinguishable from a large rock until you touch it and can feel the warm glow of potential life energy from within.  It's hard to believe that an owlbear could grow so quickly from something so small.  Perhaps you could carefully crack it open to get at the liquid golden goodness inside.",
+    "price": 300000,
+    "material": "egg",
+    "volume": 10,
+    "stack_size": 1,
+    "fun": -18,
+    "flags": [ "FREEZERBURN", "INEDIBLE", "CATTLE" ],
+    "//": "This is kludgy. A post-thresh cattle mutant who doesn't have herbivore could eat this egg but that's a pretty narrow case. The only more complete way I know to make it inedible would be to give it the powder material but then that would be more obvious to players.",
```

Comment:
this does not strictly need to be inedble: i'm fairly sure there's a way to force the player to have an item in the inventory to eat an item. Check out how serums require syringues, and require a hammer or something.

Reply:
I finally realized I could just give it the "stone" material. I'd been making a big deal out of the egg being stone-_like_ so I don't think it'll break immersion or anything.